lib/commands/web-view/list-web-views.ts

```diff
@@ -0,0 +1,31 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import helpers = require("../../common/helpers");
+
+export class ListWebViewsCommand implements ICommand {
+	constructor(private $webViewService: IWebViewService,
```

Comment:
Yes, this is the desired behavior. I think that this command is useless outside of project.